src/SetPath.py

```diff
@@ -8,7 +7,7 @@ def SetPathLoadAndSave (Geometry):
         case "Box":
             urdf_path ="./models/humanModelTemplate/humanModelTemplate_shoulder_Box.urdf"
     
-    output_file = "./models/humanModelTemplate/humanModelGenerated.urdf"
+    output_file = "./models/humanModels/" + NameFile + "humanModel.urdf"
```

Comment:
This is a bit strange. Since you are setting the `NameFile` variable to `HumanModelGenerate`, then the final file will be `HumanModelGeneratehumanModel.urdf`?

Reply:
I fixed the redundancy of the name by changing it like this:
` FileName='Subject_Name' `
and adding an underscore in the suffix:
`  '_humanModel.urdf' ` 

I fixed the problem of the missing folder by creating it if it does not already exist:
```
dirPathSave = "./models/humanModels/"
os.makedirs(dirPathSave,exist_ok=True)
```